conditions.py

```diff
@@ -31,7 +31,7 @@ def is_active(deployment_condition):
             return (datetime.datetime.now(timezone) - deployment_condition.last_update_time) <= max_inactive_time
 
         def checkdeployment(d):
-            return any(is_active(c) for c in d.status.conditions)
+            return any(is_active(c) for c in d.status.conditions where c.type == 'Progressing')
```

Comment:
is there a where clause in python? I think it's `if`

Reply:
You're right my C# is leaking in.